File: 1660-thousand-separator/thousand-separator.cpp
class Solution {
public:
    string thousandSeparator(int n) {
        if(n < 1000) return to_string(n);
        else {
            string s = "";
            int c = 0;
            while(n){ 
                if(c < 3) c++ , s.push_back((n % 10) + '0') , n /= 10;
                else s.push_back('.') , c = 0;
            }
            reverse(s.begin() , s.end());
            return s;
        }
        return "";
    }
};

File: 1660-thousand-separator/thousand-separator.py
class Solution(object):
    def thousandSeparator(self, n):
        if(n < 1000):
            return str(n)
        else :
            s = ''
            c = 0
            while(n != 0):
                if(c < 3) :
                    c += 1
                    s += (str(n % 10))
                    n = n // 10
                else :
                    s += ('.')
                    c = 0
            s = s[::-1]
            return s
        return ''